### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by vicfriedman**
issue #3 @AnnJohn

### CODE REVIEW

File: app/views/posts/new.html.erb [hunk 1]
```diff
@@ -1,8 +1,16 @@
 <%= form_tag posts_path do %>
-  <label>Post title:</label><br>
-  <%= text_field_tag :title, @post.title %><br>
+  <%= form_tag posts_path do %>
```
Comment (jmburges) [inline app/views/posts/new.html.erb]: Double `form_tag`?

**Comment by jmburges** [comment]
@vicfriedman does this solve the problem that Corey was having? Can you clone his repo and make sure these tests work against his? https://github.com/c1505/simple-partials-reading-v-000

**Comment by jmburges** [comment]
oh! I see what you're doing. You are making this lab no longer have an `@author` in the partial right? We need that to happen because the *next* lab has them feel the pain of using IVs in your partials and not just using `locals`

**Comment by vicfriedman** [comment]
closing this per conversation with @jmburges and reopening a new one

**Pull request closed without merging**